DEBUG    logger:test_cases.py:37 响应结果：
INFO     logger:test_cases.py:40 ----------------断言结束-----------------